tests/__init__.py

```diff
@@ -135,13 +135,15 @@ def boot(self, options: Sequence[str] = (), args: Sequence[str] = ()) -> Complet
         return result
 
     def vm(self, options: Sequence[str] = (), args: Sequence[str] = ()) -> CompletedProcess:
+        need_hyperv_workaround = os.uname().machine == "x86_64"
+
         result = self.mkosi(
             "vm",
             [
                 "--runtime-build-sources=no",
                 "--vsock=yes",
                 # TODO: Drop once both Hyper-V bugs are fixed in Github Actions.
```

Comment:
7a36ebeecbee5464880662d65512c14bec8cb813 originally introduced this comment. Unfortunately the commit message has no further information, do you know what the status of these bugs is, @DaanDeMeyer?